frontend/src/components/eMIB/Email.jsx

```diff
@@ -9,7 +9,15 @@ const styles = {
   },
   emailId: {
     float: "left",
-    marginRight: 12
+    marginRight: 12,
+    fontSize: 16,
+    fontWeight: 700,
+    fontFamily: '"Nunito Sans", sans-serif',
```

Comment:
Shouldn't this font be already there by default?

Reply:
This is the computed font if I do not override: 
![image](https://user-images.githubusercontent.com/2746350/55485015-97963f00-55f7-11e9-9428-e26d9f09fe4d.png)

That being said:
Nunito Sans:
![image](https://user-images.githubusercontent.com/2746350/55485114-c2809300-55f7-11e9-9de7-99196fc54010.png)

Rubick:
![image](https://user-images.githubusercontent.com/2746350/55485057-ac72d280-55f7-11e9-9875-f1fbf5e0e5c3.png)

Reply:
so the difference is actually fairly subtle. I can still remove it

Reply:
yeah I'd remove the font.

Reply:
The original H5 in aurora should be Nunito reg 18pt, but having it as Rubik med 18pt is fine too.

Reply:
font removed in 70cbccb